src/error-handlers/validation.js

```diff
@@ -0,0 +1,33 @@
+import * as Instance from "@hyperjump/json-schema/instance/experimental";
+
+/**
+ * @import { ErrorHandler, ErrorObject } from "../index.d.ts"
+ */
+
+/** @type ErrorHandler */
+// eslint-disable-next-line @typescript-eslint/require-await
+const validation = async (normalizedErrors, instance, localization) => {
+  /** @type ErrorObject[] */
+  const errors = [];
+
+  if (normalizedErrors["https://json-schema.org/validation"]) {
+    for (const schemaLocation in normalizedErrors["https://json-schema.org/validation"]) {
+      // additionalProperties has its own specific error handler; avoid duplicate messages
+      if (schemaLocation.endsWith("/additionalProperties")) {
+        continue;
+      }
+      const value = normalizedErrors["https://json-schema.org/validation"][schemaLocation];
+      if (value === false) {
```

Comment:
This needs a better name. Try,


```suggestion
      const isValid = normalizedErrors["https://json-schema.org/validation"][schemaLocation];
      if (!isValid) {
```